platform_versioned_docs/version-24.1.1/getting-started/quickstart-demo/automation.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: "Automation"
+description: "An introduction to automation with APIs and CLI tools in Seqera Platform"
+date: "21 Jul 2024"
+tags: [platform, automation, api, cli, seqerakit]
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Seqera Platform provides multiple methods of programmatic interaction to automate the execution of pipelines, chain pipelines together, and integrate Platform with third-party services.
```

Comment:
Chaining pipelines together is still a little hacky. I'm not sure if it should be promoted here.